src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutesSpec.scala

```diff
@@ -51,21 +51,17 @@ class LeoRoutesSpec extends FlatSpec with ScalatestRouteTest with CommonTestData
 
   it should "200 when creating and getting cluster" in isolatedDbTest {
     val newCluster = ClusterRequest(Map.empty, Some(jupyterExtensionUri), Some(jupyterUserScriptUri), None, None, Some(UserJupyterExtensionConfig(Map("abc" ->"def"))))
-    val clusterNames = Seq(clusterName.value, s"${clusterName.value}-v2")
-    val baseUrl = "/api/cluster"
-    val versionedBaseUrls = Seq(baseUrl, s"$baseUrl/v2")
-
-    // PUT endpoints have two versions
-    versionedBaseUrls.zip(clusterNames) foreach { case (url, cn) =>
-      Put(s"$url/${googleProject.value}/$cn", newCluster.toJson) ~>
-        timedLeoRoutes.route ~> check {
-          status shouldEqual StatusCodes.OK
-          validateCookie { header[`Set-Cookie`] }
+
+    forallClusterCreationVersions(clusterName) { (version, clstrName, statusCode) =>
```

Comment:
Using `forallClusterCreationVersions` allowed for keeping the testing logic the same and prevented code duplication.